### DISCUSSION TITLE
Support Chakra V3

### DISCUSSION THREAD

**Discussion by abrenoch** [csandman/chakra-react-select#345] [Ideas]
[Version 3 of chakraUI was recently released](https://www.chakra-ui.com/docs/get-started/migration) - would love to se support added for it!

**Comment by csandman** [comment]
Based on the fact that you closed this, I assume you found the discussion that's already going about Chakra V3. But if not, there is ongoing discussion in #319

**Reply by abrenoch** [reply]
Hey yea I used the link from the issue template page and didn't notice it was filtering the threads when I glanced for an existing one. Found it immediately after posting this and felt like a silly goose. Sorry about that!
